### ISSUE TITLE
PyInstaller packages PySide2 not correctly on Ubuntu

### ISSUE THREAD

**Issue by CaptainDario** [pyinstaller/pyinstaller#5462]
## Context information (for bug reports)

* Output of `pyinstaller --version`: ```4.1```
* Version of Python: 3.8.5
* Platform: Linux (Ubuntu 20.4)
* I already packaged the application successfully on Windows 10 without this bug.

## Bug description
My application uses QML for the ui.
If I package the application on Ubuntu 20.4 the graphics of my buttons move to the top left corner. But the actual button (the part which reacts to clicking) stays in the correct position.
However if I copy the whole ```PySide2```-folder from the ```site-packages```-folder to the folder made by pyinstaller it works correctly.
Also packaging on windows works correctly.

## PyInstaller output
```
pyinstaller --add-data ./ui:ui --add-data ./data:data --add-data ./icons:icons  --distpath=./build/Linux --name=DaKanjiRecognizer_v1.0_folder --clean --icon=./icons/icon.ico --noconfirm ./src/main.py
27 INFO: PyInstaller: 4.1
27 INFO: Python: 3.8.5
36 INFO: Platform: Linux-5.4.0-42-generic-x86_64-with-glibc2.29
36 INFO: wrote /home/dario/dev/DaKanjiRecognizer/DaKanjiRecognizer_v1.0_folder.spec
38 INFO: UPX is not available.
39 INFO: Removing temporary files and cleaning cache in /home/dario/.cache/pyinstaller
39 INFO: Extending PYTHONPATH with paths
['/home/dario/dev/DaKanjiRecognizer/src', '/home/dario/dev/DaKanjiRecognizer']
42 INFO: checking Analysis
44 INFO: Building Analysis because Analysis-00.toc is non existent
44 INFO: Initializing module dependency graph...
44 INFO: Caching module graph hooks...
51 INFO: Analyzing base_library.zip ...
1916 INFO: Processing pre-find module path hook distutils from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/pre_find_module_path/hook-distutils.py'.
1916 INFO: distutils: retargeting to non-venv dir '/usr/lib/python3.8'
3669 INFO: Caching module dependency graph...
3763 INFO: running Analysis Analysis-00.toc
3795 INFO: Analyzing /home/dario/dev/DaKanjiRecognizer/src/main.py
4306 INFO: Processing pre-safe import module hook setuptools.extern.six.moves from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/pre_safe_import_module/hook-setuptools.extern.six.moves.py'.
4535 INFO: Processing pre-find module path hook site from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/pre_find_module_path/hook-site.py'.
4536 INFO: site: retargeting to fake-dir '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/fake-modules'
10212 INFO: Processing module hooks...
10213 INFO: Loading module hook 'hook-sklearn.cluster.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
10214 INFO: Loading module hook 'hook-sklearn.metrics.cluster.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
10216 WARNING: Hidden import "sklearn.utils.lgamma" not found!
10216 WARNING: Hidden import "sklearn.utils.weight_vector" not found!
10216 INFO: Loading module hook 'hook-sklearn.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
10326 INFO: Loading module hook 'hook-sklearn.utils.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
10328 INFO: Loading module hook 'hook-sklearn.neighbors.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
10328 WARNING: Hidden import "sklearn.neighbors.typedefs" not found!
10329 WARNING: Hidden import "sklearn.neighbors.quad_tree" not found!
10372 INFO: Loading module hook 'hook-encodings.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10413 INFO: Loading module hook 'hook-numpy.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10414 INFO: Loading module hook 'hook-scipy.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10415 INFO: Loading module hook 'hook-sysconfig.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10426 INFO: Loading module hook 'hook-numpy.core.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10545 INFO: Loading module hook 'hook-heapq.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10548 INFO: Excluding import of doctest from module heapq
10548 INFO: Loading module hook 'hook-distutils.util.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10549 INFO: Excluding import of lib2to3.refactor from module distutils.util
10550 INFO: Loading module hook 'hook-distutils.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10550 INFO: Loading module hook 'hook-lib2to3.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10559 INFO: Loading module hook 'hook-scipy.stats._stats.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10560 INFO: Loading module hook 'hook-PIL.ImageFilter.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10562 INFO: Excluding import of numpy from module PIL.ImageFilter
10562 INFO: Loading module hook 'hook-scipy.special._ellip_harm_2.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10563 INFO: Loading module hook 'hook-scipy.special._ufuncs.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10564 INFO: Loading module hook 'hook-PySide2.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10740 INFO: Loading module hook 'hook-PIL.Image.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10948 INFO: Loading module hook 'hook-multiprocessing.util.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
10952 INFO: Excluding import of test.support from module multiprocessing.util
10952 INFO: Excluding import of test from module multiprocessing.util
10952 INFO: Loading module hook 'hook-PySide2.QtGui.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11046 INFO: Loading module hook 'hook-xml.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11082 INFO: Loading module hook 'hook-numpy._pytesttester.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11087 INFO: Excluding import of pytest from module numpy._pytesttester
11087 INFO: Loading module hook 'hook-PySide2.QtNetwork.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11157 INFO: Loading module hook 'hook-difflib.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11162 INFO: Excluding import of doctest from module difflib
11162 INFO: Loading module hook 'hook-PySide2.QtCore.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11237 INFO: Loading module hook 'hook-pickle.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11241 INFO: Excluding import of argparse from module pickle
11241 INFO: Loading module hook 'hook-scipy.linalg.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11242 INFO: Loading module hook 'hook-xml.etree.cElementTree.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11242 INFO: Loading module hook 'hook-pkg_resources.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11551 INFO: Processing pre-safe import module hook win32com from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/pre_safe_import_module/hook-win32com.py'.
11553 WARNING: Hidden import "pkg_resources.py2_warn" not found!
11554 WARNING: Hidden import "pkg_resources.markers" not found!
11557 INFO: Excluding import of __main__ from module pkg_resources
11557 INFO: Loading module hook 'hook-scipy.sparse.csgraph.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11559 INFO: Loading module hook 'hook-PySide2.QtQml.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11683 INFO: Loading module hook 'hook-PySide2.QtWidgets.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11792 INFO: Loading module hook 'hook-PIL.SpiderImagePlugin.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11795 INFO: Import to be excluded not found: 'FixTk'
11796 INFO: Loading module hook 'hook-PIL.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
11798 INFO: Excluding import of PyQt5 from module PIL.ImageQt
11800 INFO: Import to be excluded not found: 'FixTk'
11802 INFO: Excluding import of tkinter from module PIL.ImageTk
11803 INFO: Loading module hook 'hook-setuptools.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
12146 INFO: Excluding import of setuptools.py33compat from module setuptools.depends
12147 INFO: Excluding import of setuptools.py33compat from module setuptools.package_index
12148 INFO: Excluding import of setuptools.py27compat from module setuptools.depends
12149 INFO: Excluding import of setuptools.py27compat from module setuptools.command.easy_install
12149 INFO: Excluding import of setuptools.py27compat from module setuptools.package_index
12150 INFO: Loading module hook 'hook-sklearn.tree.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/_pyinstaller_hooks_contrib/hooks/stdhooks'...
12151 INFO: Loading module hook 'hook-setuptools.msvc.py' from '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks'...
12153 INFO: Excluding import of numpy from module setuptools.msvc
12184 INFO: Looking for ctypes DLLs
12422 WARNING: library {}.dll required via ctypes not found
12431 INFO: Analyzing run-time hooks ...
12444 INFO: Including run-time hook '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/rthooks/pyi_rth_multiprocessing.py'
12447 INFO: Including run-time hook '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/rthooks/pyi_rth_pkgres.py'
12448 INFO: Including run-time hook '/home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/hooks/rthooks/pyi_rth_pyside2.py'
12467 INFO: Looking for dynamic libraries
17844 INFO: Looking for eggs
17844 INFO: Python library not in binary dependencies. Doing additional searching...
17865 INFO: Using Python library /lib/x86_64-linux-gnu/libpython3.8.so.1.0
17880 INFO: Warnings written to /home/dario/dev/DaKanjiRecognizer/build/DaKanjiRecognizer_v1.0_folder/warn-DaKanjiRecognizer_v1.0_folder.txt
17982 INFO: Graph cross-reference written to /home/dario/dev/DaKanjiRecognizer/build/DaKanjiRecognizer_v1.0_folder/xref-DaKanjiRecognizer_v1.0_folder.html
18068 INFO: Appending 'datas' from .spec
18071 INFO: checking PYZ
18072 INFO: Building PYZ because PYZ-00.toc is non existent
18072 INFO: Building PYZ (ZlibArchive) /home/dario/dev/DaKanjiRecognizer/build/DaKanjiRecognizer_v1.0_folder/PYZ-00.pyz
19144 INFO: Building PYZ (ZlibArchive) /home/dario/dev/DaKanjiRecognizer/build/DaKanjiRecognizer_v1.0_folder/PYZ-00.pyz completed successfully.
19172 INFO: checking PKG
19172 INFO: Building PKG because PKG-00.toc is non existent
19172 INFO: Building PKG (CArchive) PKG-00.pkg
19209 INFO: Building PKG (CArchive) PKG-00.pkg completed successfully.
19211 INFO: Bootloader /home/dario/dev/DaKanjiRecognizer/.venv_rel/lib/python3.8/site-packages/PyInstaller/bootloader/Linux-64bit/run
19211 INFO: checking EXE
19211 INFO: Building EXE because EXE-00.toc is non existent
19211 INFO: Building EXE from EXE-00.toc
19211 INFO: Appending archive to ELF section in EXE /home/dario/dev/DaKanjiRecognizer/build/DaKanjiRecognizer_v1.0_folder/DaKanjiRecognizer_v1.0_folder
19234 INFO: Building EXE from EXE-00.toc completed successfully.
19237 INFO: checking COLLECT
19237 INFO: Building COLLECT because COLLECT-00.toc is non existent
19237 INFO: Removing dir ./build/Linux/DaKanjiRecognizer_v1.0_folder
19324 INFO: Building COLLECT COLLECT-00.toc
19812 INFO: Building COLLECT COLLECT-00.toc completed successfully.
pyinstaller --add-data ./ui:ui --add-data ./data:data --add-data ./icons:icons  --distpath=./build/Linux --name=DaKanjiRecognizer_v1.0_file --clean --onefile --icon=./icons/icon.ico --noconfirm ./src/main.py

```

### A minimal example program which shows the error
Maybe this is not a minimal example but my application with detailed setup information can be found on the [```feature-linux_support```-branch](https://github.com/CaptainDario/DaKanjiRecognizer/tree/Feature-linux_support).
The pyinstaller script can be found in ```build.py````.

### Stacktrace / full error message
```
The program runs and there are no errors. However the UI is broken.
```

**Comment by rokm** [comment]
This could be due to missing `xcbglintegrations` plugins, and should be fixed in `develop` branch since #5349. Can you try that one? (Or, just manually copy only `xcbglintegrations` plugins to confirm that their absence is causing the issue).

**Comment by CaptainDario** [comment]
Thank you for the quick reply!
Indeed adding ```--add-data/plugins/xcbglintegrations:/xcbglintegrations``` solved the problem for me!

Thank you a lot!
